<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKUdpServerMulticast</class>
 <widget class="QDialog" name="SAKUdpServerMulticast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP Server Multicast Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="applyPushButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="tipsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: &quot;red&quot;;
}</string>
     </property>
     <property name="text">
      <string notr="true"> </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SAKUdpServerMulticast</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
